packages/portal-proto/src/pages/annotations/index.tsx

```diff
@@ -10,6 +11,10 @@ const AnnotationsPage: NextPage = () => {
     selectGdcAppById(AnnotationBrowserAppId),
   ) as React.ElementType;
 
+  datadogRum.startView({
+    name: "Annotation",
```

Comment:
nitpick:  could we make this plural, `Annotation Browser`?

Reply:
Done 👍